tekton-pipelines/base/tasks/slack-notification-task.yaml

```diff
@@ -52,6 +101,12 @@ spec:
 
         echo "result: $status"
 
+        thread_env_file="$(workspaces.source.path)/env/SLACK_THREAD"
```

Comment:
use outputs for task instead of env vars.
See docs: https://tekton.dev/docs/pipelines/tasks/#emitting-results